lib/client.js

```diff
@@ -794,6 +798,10 @@ function resume (client) {
     }
 
     if (!client.pending) {
+      if (client.connected && !client[kWriting] && !client[kReset]) {
+        // TODO: Only emit if was busy.
+        client.emit('drain')
```

Comment:
I'm a little unsure about the naming here... Also it might be emitted multiple times, though that is probably mostly harmless.

Reply:
drain is fine!